vault.js

```diff
@@ -1,4 +1,21 @@
 'use strict';
 module.exports = function() {
   
+    let vault = {key: null, value: null}
```

Comment:
So an object already have a way to keep track of keys and value. Rather than creating an object with a "key" key and a "value" key. You could just use: `{ key: value }`